test/function-curry/index.js

```diff
@@ -2,30 +2,35 @@ var test = require('tape');
 var curry = require('../../packages/function-curry');
 
 test('binds curried arguments to supplied arguments', function (t) {
-  t.plan(4);
+  t.plan(3);
   function converter(ratio, input) {
     return (input * ratio).toFixed(1);
   }
   var milesToKm = curry(converter, 1.62);
```

Comment:
I think we should make this no longer work.
You just curry with no args except the fn